scripts/start-dependencies.sh

```diff
@@ -85,3 +85,8 @@ if [ "$INCLUDE_E2E" = true ]; then
   ./scripts/seed-infrastructure-links.sh testdb-e2e &
 fi
 wait
+./scripts/seed-municipalities-and-fare-zones.sh 3010 &
```

Comment:
Please add a comment into which server this seed data is intended to be loaded into. The magic port number `3010` itse does not tell much to the reader.